go.mod

```diff
@@ -4,34 +4,33 @@ go 1.13
 
```

Comment:
Should we do go 1.14 here above?

Reply:
I don't think we have any features in CMO which need 1.14 and codebase can be compiled with 1.13, so the current setting seems good to me.